pkgs/top-level/python-packages.nix

```diff
@@ -841,6 +841,8 @@ in {
 
   phonopy = callPackage ../development/python-modules/phonopy { };
 
+  phik = callPackage ../development/python-modules/phik {};
```

Comment:
If I read your expression, and particularly the `fetchPyPi` section, it seems that this package is only for `Python3`.
Then, better disable in other cases:
```
phik = disabledIf (!isPy3k) (callPackage ../development/python-modules/phik {                                                                           
  inherit (pkgs) pkgconfig;                                                                                                                                       
});
```

That would not allow users to install `python2Packages.phik` .

Reply:
it should be moved into the expression it self, just a `disabled = !isPy3k` should suffice